src/store/flow/block/choice.ts

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable max-len */
+
+import {IRootState} from '@/store'
+import {
+  findBlockWith, IBlock,
+  IChoice,
+  IFlow, ILanguage, IResource,
+  ISelectOneResponseBlock,
+  ValidationException,
+} from '@floip/flow-runner'
+import {ActionTree, GetterTree, MutationTree} from 'vuex'
+import {deleteChoiceValueByPath} from '@/store/flow/utils/vuexBlockHelpers'
+import {IEmptyState} from '@/store/flow/block-types/BaseBlock'
+import Vue from 'vue'
+
+export const BLOCK_RESPONSE_EXPRESSION = 'block.response'
+
+export function textValueToExpression(value: string): string {
+  if (value.includes('@') === true) {
+    return value
+  } else {
+    // Do not accept single quote character for this scenario to make it simple
+    return `@block.response = '${value.replace(/'/g, '')}'`
```

Comment:
I wonder if escaping with a `\` is what the grammar of the flow-parser allows:
https://github.com/FLOIP/expression-parser/blob/master/src/pegjs/floip.pegjs#L329-L333